### DISCUSSION TITLE
How to send strings to a remote file?

### DISCUSSION THREAD

**Discussion by rodrigomologni** [ParallelSSH/parallel-ssh#321] [Q&A] [answered]
What is the method equivalent to the [`paramiko.sftp_client.SFTPClient.open`](http://docs.paramiko.org/en/stable/api/sftp.html#paramiko.sftp_client.SFTPClient.open) in **ParallelSSH**? I would like to open/create a remote file and to send strings to this file.

**Comment by pkittenis** [comment, marked as answer]
Currently the clients support copying local files. Opening remote file functions are not public, so will need to write to a temporary local file and copy that via the existing methods.

See [copy_file](https://parallel-ssh.readthedocs.io/en/latest/native_parallel.html#pssh.clients.native.parallel.ParallelSSHClient.copy_file) for copy functionality.

There is an [open issue to make them public](https://github.com/ParallelSSH/parallel-ssh/issues/117), but not high priority atm.

**Reply by rodrigomologni** [reply]
@pkittenis thank you!
